src/GregClientTests/GregClientTests.csproj

```diff
@@ -10,6 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="JunitXml.TestLogger" Version="3.0.134" />
```

Comment:
Why is this needed now?

Reply:
I've added the tests to the pipeline and need to use this logger (we use it in Dynamo) for compatibility with ci library.